### PR TITLE
Fixes #8314 for realsies this time

### LINKED ISSUE #8314: Treadmills take less than a minute to bring you to 100Cº, at which point, they still give you massive burn damage and also exploding knees
stock-vector-rubber-stamp-with-text-one-hundred-percent-tested-confirmed-inside-vector-illustration-287448125

### PR DESCRIPTION
**Pull request by ghost**
- Extensively tested (tm)
- Adds new emagged functionality to treadmills
- Fixes #8314
- makes burn_calories more OOP sorta
- adds cap to burn_calories for now

### CODE REVIEW

File: code/modules/power/treadmill.dm [hunk 1]
```diff
@@ -95,3 +98,9 @@
 	..()
 	if(anchored) connect_to_network()
 	else disconnect_from_network()
+
+/obj/machinery/power/treadmill/emag()
+	..()
+	emagged = 1
+	name = "DREADMILL"
```
Comment (9600bauds) [inline code/modules/power/treadmill.dm]: :+1::x::100:

**Comment by NigglyWiggly** [comment]
The Dreadmill moves on its own, I hope?

If not it's kind of pointless, as the crazy mass killing was only happening because of the conveyor belts I added. It's still dangerous, but not nearly as insane

**Comment by Imagebard** [comment]
Okay, so all you need is a emag and a treadmill to spawn a godkiller?

**Comment by 9600bauds** [comment]
You can kill gods in a multitude of ways, literally just get them on a table and target mouth with a sharp object

**Comment by ghost** [comment]
Or run at them with an axe aimed for the mouth

File: code/modules/power/treadmill.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/power/treadmill.dm]: Needs improper though.

**Pull request merged**